src/Sylius/Behat/Service/ElasticsearchChecker.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service;
+
+use FOS\ElasticaBundle\Elastica\Client;
+
+/**
+ * @author Nicolas Adler <nicolas.adler@openizi.com>
+ */
+final class ElasticsearchChecker implements ElasticsearchCheckerInterface
```

Comment:
I would change the name of this service to something like `ElasticSearchRefresher` WDYT?